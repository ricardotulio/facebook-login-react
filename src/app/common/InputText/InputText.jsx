import React from 'react'

import Input from '../Input'

const InputText = (props) => {
  const typeText = {
    ...props,
    type: "text",
  }

  return (
    <Input {...typeText} />
  )
}

export default InputText