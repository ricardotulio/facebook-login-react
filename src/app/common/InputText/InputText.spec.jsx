import React from 'react'
import renderer from 'react-test-renderer'

import InputText from './InputText'

test('Properties are passed element', () => {
  const inputText = renderer.create(
    <InputText id="inputText-id" name="inputText-name" />
  )
  let tree = inputText.toJSON()

  expect(tree).toMatchSnapshot()
})

test('Type property cannot be overwritten', () => {
  const inputText = renderer.create(
    <InputText id="inputText-id" name="inputText-name" type="password" />
  )
  let tree = inputText.toJSON()

  expect(tree).toMatchSnapshot()
})