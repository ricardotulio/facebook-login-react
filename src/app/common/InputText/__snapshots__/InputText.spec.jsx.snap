// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Properties are passed element 1`] = `
<input
  id="inputText-id"
  name="inputText-name"
  type="text"
/>
`;

exports[`Type property cannot be overwritten 1`] = `
<input
  id="inputText-id"
  name="inputText-name"
  type="text"
/>
`;
